docs/release-notes/0.10.2.md

```diff
@@ -4,7 +4,7 @@
 #### Bug fixes
 
 * Added compatibility layer for packages relying on `anndata._core.sparse_dataset.SparseDataset`.
-  Note that this API is *deprecated* and new code should use {class}`~anndata.abc.CSRDataset`, {class}`~anndata.abc.CSCDataset`, and {func}`~anndata.sparse_dataset` instead.
+  Note that this API is *deprecated* and new code should use {class}`~anndata.abc.CSRDataset`, {class}`~anndata.abc.CSCDataset`, and {func}`~anndata.io.sparse_dataset` instead.
```

Comment:
I don't think we should change the text of old release notes. But I do think it's fine if the links don't work anymore. So instead I'd just do:

```suggestion
  Note that this API is *deprecated* and new code should use `anndata.CSRDataset`, `~anndata.CSCDataset`, and `anndata.sparse_dataset` instead.
```

Reply:
For the future, that’s less work. But Ilan already did the work, so I don’t think it needs to be undone.